modules/ipcCommunicator.js

```diff
@@ -34,7 +35,7 @@ ipc.on('backendAction_closeApp', function() {
 ipc.on('backendAction_closePopupWindow', function(e) {
     var windowId = e.sender.getId();
 
-    if(global.windows[windowId]) {
+    if(_.get(global.windows, windowId)) {
```

Comment:
I didn't know about `_.get()` seems like a neat function. Though for this use case its probably a bit to much ;)